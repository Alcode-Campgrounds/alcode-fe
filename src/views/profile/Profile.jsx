
export default function Profile() {
    return (
       <> 
            <h2>This is your life</h2>
            <div className='fav-list-parent'>
                <ul className='fav-list'>
                    <li>your favorite 1</li>
                    <li>your favorite 2</li>
                    <li>your favorite 3</li>
                </ul>
            </div>
            
        </>
        )
    }
