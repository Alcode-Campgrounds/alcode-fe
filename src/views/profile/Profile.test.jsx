import React from 'react'
import renderer from 'react-test-renderer'
import Profile from './Profile'



describe('is it here?', () => {
    
    const wrapper = renderer.create(<Profile />)

    test('did it render?', () => {
        expect(wrapper).toMatchSnapshot()
    })
})
