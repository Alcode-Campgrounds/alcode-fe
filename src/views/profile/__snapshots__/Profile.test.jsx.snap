// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is it here? did it render? 1`] = `
Array [
  <h2>
    This is your life
  </h2>,
  <div
    className="fav-list-parent"
  >
    <ul
      className="fav-list"
    >
      <li>
        your favorite 1
      </li>
      <li>
        your favorite 2
      </li>
      <li>
        your favorite 3
      </li>
    </ul>
  </div>,
]
`;
